### ISSUE TITLE
When selection drag is over, can I keep the lasso box?

### ISSUE THREAD

**Issue by NewDolphin** [skokenes/d3-lasso#12]
When selection drag is over, can I keep the lasso box?

**Comment by gioele-antoci** [comment]
Not with how the code has been built. You will have to modify this part here: 

https://github.com/skokenes/d3-lasso/blob/master/src/lasso.js#L162

**Comment by nateEaston** [comment]
Yeah, I am kind of thinking the same thing. I want to implement a multiple selection when the 'ctrl' key is pressed.
